src/manuel/tests/test_module.py

```diff
@@ -79,6 +80,31 @@ def test_suite():
     m += manuel.testcase.SectionManuel()
     # The apparently redundant "**dict()" is to make this code compatible with
     # Python 2.5 -- it would generate a SyntaxError otherwise.
+    suite = manuel.testing.TestSuite(
+        m, *tests, **dict(globs={'path_to_test': os.path.join(here, 'doctests/rst/bugs.txt')})
+    )
+
+    return unittest.TestSuite(
+        (
+            suite,
+            doctest.DocTestSuite(),
+        )
+    )
+
+
+def test_suite_myst():
+    tests = ['doctests/myst/myst-markdown.md']
+    print("*** test_suite_myst. tests", tests)
+
+    optionflags = doctest.NORMALIZE_WHITESPACE | doctest.ELLIPSIS
+
+    # m = manuel.ignore.Manuel()
+    m = manuel.doctest.Manuel(optionflags=optionflags, checker=checker)
+    m += manuel.myst.codeblock.Manuel()
+    # m += manuel.capture.Manuel()
+    # m += manuel.testcase.SectionManuel()
+    # The apparently redundant "**dict()" is to make this code compatible with
+    # Python 2.5 -- it would generate a SyntaxError otherwise.
```

Comment:
I'm happy to drop support for any Python that has [reached end-of-life](https://devguide.python.org/#status-of-python-branches).

Reply:
I'm willing to create a PR to update the dependencies and supported Python versions in a separate PR. OK?